assets/src/edit-story/utils/localStore.js

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Moved our localStorage calls to one file to wrap the calls to get/set around general functionality we always want to do (like wrapping `getItem` for `try/catch` and stuff).